src/fidesops/util/cryptographic_util.py

```diff
@@ -11,11 +14,22 @@ def hash_with_salt(text: bytes, salt: bytes) -> str:
     return hashlib.sha512(text + salt).hexdigest()
 
 
-def generate_secure_random_string(length_in_bytes: int) -> str:
-    """Generates a securely random string using Python secrets library"""
-    return secrets.token_hex(length_in_bytes)
+def generate_secure_random_string(length: int) -> str:
+    """Generates a securely random string using Python secrets library
+    that is twice the length of the specified input"""
+    return secrets.token_hex(length)
 
 
 def generate_salt() -> str:
     """Generates a salt using bcrypt and returns a string using the configured default encoding"""
     return bcrypt.gensalt().decode(config.security.ENCODING)
+
+
+def bytes_to_str(bytestring: bytes) -> str:
```

Comment:
I was a bit surprised that this returned a b64-encoded string, considering that Python allows you to just tuck a `bytes` array into a `str` instance whenever you like.

For that reason, maybe consider calling these helpers `bytes_to_b64_str` and `b64_str_to_bytes`?

Reply:
oh yes that's much more clear